File: src/main/java/com/hank/lottery/Game539.java
package com.hank.lottery;

public class Game539 extends NumberGame{
    int numberCount = 5;

    @Override
    public void generate() {
        super.numberCount = numberCount;
        numbers.add(11);
        numbers.add(22);
        numbers.add(33);
        numbers.add(44);
        numbers.add(55);
        System.out.println("樂透539 : set長度 " + numbers.size());
        System.out.println("需求長度是 " + numberCount +"：" + validate());
    }
}

File: src/main/kotlin/com/hank/lottery/Game539.kt
package com.hank.kotlin

class Game539:NumberGame() {

    override fun generate() {
        val numberCount539:Int = 5
        super.numberCount = numberCount539
        numbers.add(100)
        numbers.add(200)
        numbers.add(300)
        numbers.add(400)
        println("kotlin Game539 set長度=${numbers.size}")
        println("kotlin Game539 需求長度=$numberCount539")
        println("kotlin Game539 選號數量是否正常=${validate()}")
    }
}